MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,373 @@
+
+The data form control automatically generates `SfDataForm.Items` (which has UI settings of data field) based on the data type in the `SfDataForm.DataObject` property. The `SfDataForm.Items` encapsulates the layout of label and editor setting for the data field appearing in the dataform. 
+
+The type of input editor generated for the data field depends on the type and attribute settings of the property. The following table lists the `DataFormItem` and its constraints for generation:
+
+<table>
+<tr>
+<th>Generated DataFormItem Type</th>
+<th>Data Type / Attribute</th>
+<th>Editor</th>
+<th>Input Control</th>
+</tr>
+<tr>
+<td>
+{{'[DataFormTextItem]
+`Link`
+</td>
+<td>
+Default DataFormItem generated for the String type and the properties with [DataType(DataType.Text)].
+</td>
+<td>
+Text
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormMultilineTextItem]
+`Link`
+</td>
+<td>
+Generated for string type property with [DataType(DataType.MultilineText)] attributes.
+</td>
+<td>
+Multiline Text
+</td>
+<td>
+{{'[Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormPasswordTextItem]
+`Link`
+</td>
+<td>
+Generated for string type property with [DataType(DataType.Password)] attributes.
+</td>
+<td>
+Password
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormCheckBoxItem]`Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+<td>
+CheckBox
+</td>
+<td>
+{{'[CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormSwitchItem] `Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+<td>
+Switch
+</td>
+<td>
+{{'[Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormDateItem] `Link`
+</td>
+<td>
+Generated for the DateTime, DateOnly, DateTimeOffset type properties.
+[DataType(DataType.Date)].
+[DataType(DataType.DateTime)].
+</td>
+<td>
+DatePiker
+</td>
+<td>
+{{'[DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormTimeItem]
+`Link`
+</td>
+<td>
+Generated for the TimeSpan, TimeOnly, DateTime, DateTimeOffset type properties.
+[DataType(DataType.Time)].
+</td>
+<td>
+TimePicker
+</td>
+<td>
+{{'[TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormPickerItem] `Link`
```

Comment:
'[DataFormPickerItem]'

Remove links, we can inlude after api publish, do it for all editors

Reply:
removed